int64 id
int64 object_size

